cogs/greetings.py

```diff
@@ -3,22 +3,26 @@
 import os
 from discord.ext import commands
 
+# Class dedicated to greeting new members when joining the server
 class Greetings(commands.Cog):
+
+# Constructor that assigns the welcome message to the specific new member with their username
     def __init__(self, client):
         self.client = client
         self.welcome_message_id = self.load_welcome_message_id()
-
+# Helper function responsible for loading and returning the welcome message
     def load_welcome_message_id(self):
         try:
             with open("txtfiles/welcome_message_id.txt", "r") as file:
                 return int(file.read())
         except FileNotFoundError:
             return None
-
+# Helper function responsible for retaining the welcome message specific to the user
     def save_welcome_message_id(self, message_id):
         with open("txtfiles/welcome_message_id.txt", "w") as file:
             file.write(str(message_id))
-
+# Command function responsible for running the welcome message on user join where they must react to a
```

Comment:
Suggest terminology like sending instead of running.

Reply:
Okay let me know if there is any other terminology I can use as well for other areas.